cli/kata-check_ppc64le.go

```diff
@@ -17,6 +19,11 @@ const (
 	archCPUModelField  = genericCPUModelField
 )
 
+var (
+	ppc64CpuCmd     = "ppc64_cpu"
+	smtStatusOption = "--smt"
```

Comment:
Is this option guaranteed to be available in all versions of `ppc64_cpu`?

Reply:
It's existed since the initial version AFAICS (~2009), and would obviously never be removed.